forest/sycamore/submits.py

```diff
@@ -332,44 +338,45 @@ def survey_submits(
         np.array("NaT", dtype="datetime64[ns]")
     )
 
+    # mark as NA if no submit happened
     submit_lines3["time_to_submit"] = np.where(
         submit_lines3["submit_flg"] == 1,
         submit_lines3["time_to_submit"],
-        np.array("NaT", dtype="datetime64[ns]")
+        np.NaN
     )
 
     return submit_lines3.sort_values(
         ["survey id", "beiwe_id"]
     ).drop_duplicates(), submit_lines_summary
 
 
-def survey_submits_no_config(study_dir: str,
-                             study_tz: str = "UTC") -> pd.DataFrame:
+def survey_submits_no_config(input_agg: pd.DataFrame) -> pd.DataFrame:
     """Get survey submits without config file
 
     Alternative function for getting the survey completions (doesn't have
     expected times of surveys)
     Args:
-        study_dir(str):
-            Directory where information is located
-        study_tz(str):
-            Time zone for local time
+        input_agg(DataFrame):
+            Dataframe of Aggregated Data
+    Returns:
+        Dataframe with one line per survey submission.
+
     """
-    tmp = aggregate_surveys_no_config(study_dir, study_tz)
+    agg = input_agg.copy()
```

Comment:
Is copying actually necessary here?

Reply:
Yes, because we want to be able to write agg to a csv after this function in main. 

This function creates many new columns in agg that aren't necessary to write.